### ISSUE TITLE
Help getting a simple API example working please

### ISSUE THREAD

**Issue by e-gineer** [planetscale/planetscale-go#105]
The example in the README uses an access token, but I cannot find how to create one of those?

So, instead I'm trying to use the service token.

But, then I'm getting errors:
* `Regions.List` gives `Requires Authentication`
* `Organizations.ListRegions` gives `Not Found`

I'm stuck and obviously doing something silly / wrong. Can you please help explain how to get a simple API example working?

BTW, my ultimate goal is to create a [Steampipe plugin](https://hub.steampipe.io/plugins).

Thanks!

Here is my code:
```
package main                                                                                                                                                                             
  
import (
  "context"
  "fmt"
    
  "github.com/planetscale/planetscale-go/planetscale"
) 
    
func main() {
  
  ctx := context.Background()
    
  // create a new PlanetScale API client with the given access token
  client, err := planetscale.NewClient(
    planetscale.WithServiceToken("nw-test-2", "pscale_tkn_SECRET"),
  )
  if err != nil {
    panic(err)
  } 
  
  // Error: Requires authentication
  regions, err := client.Regions.List(ctx, &planetscale.ListRegionsRequest{})
  if err != nil {
    panic(err)
  }
  fmt.Printf("Found %d regions\n", len(regions))
  for _, i := range regions {
    fmt.Printf("Name: %q\n", i.Name)
    fmt.Printf("Location: %q\n", i.Location)
  }
    
  // Error: Not Found
  orgRegions, err := client.Organizations.ListRegions(ctx, &planetscale.ListOrganizationRegionsRequest{
    Organization: "steampipe",
  })
  if err != nil {
    panic(err)
  }
  fmt.Printf("Found %d orgRegions\n", len(orgRegions))
  for _, i := range orgRegions {
    fmt.Printf("Name: %q\n", i.Name)
    fmt.Printf("Location: %q\n", i.Location)
  }
}
```

**Comment by e-gineer** [comment]
Any chance @iheanyi or @dbussink could help with this one?

**Comment by e-gineer** [comment]
I believe that PlanetScale has both access tokens and service tokens. It seems that service tokens are more appropriate for API use, but per above I can't work out how to use them?

Anyway, #4 shows how to obtain an access token when using the PlanetScale CLI, so while that feels like a hack it works and I've moved forward with that. So, closing this (unresolved) ticket.

I've released a PlanetScale plugin for Steampipe - so users can query their PlanetScale configuration, logs, etc with SQL. Hope it's useful for the community!

Docs - https://hub.steampipe.io/plugins/turbot/planetscale

Source - https://github.com/turbot/steampipe-plugin-planetscale
